README.md

```diff
@@ -68,12 +68,12 @@ To bootstrap the necessary files and basic configuration for a new Apostrophe pi
 apos add piece vegetables
 ```
 
-Then remember to register `'vegetables': {}` in `apps.js` above.
+Then remember to register `'vegetables': {}` in `app.js` above.
```

Comment:
At this point I this line is where "vegetables" is the typo. A3 syntax is singular.